client/clients/tso/client.go

```diff
@@ -198,11 +198,12 @@ func (c *Cli) getTSOLeaderClientConn() (*grpc.ClientConn, string) {
 	if len(url) == 0 {
 		log.Fatal("[tso] the tso leader should exist")
 	}
-	cc, ok := c.svcDiscovery.GetClientConns().Load(url)
-	if !ok {
-		return nil, url
+	cc, err := c.svcDiscovery.GetOrCreateGRPCConn(url)
```

Comment:
https://github.com/tikv/pd/blob/f86063f6063813ddd6a2ec23d5b645170049b722/client/clients/router/client.go#L333-L337
I think this code also needs to change